knimin/lib/data_access.py

```diff
@@ -1386,7 +1405,10 @@ def addGeocodingInfo(self, limit=None, retry=False):
         # clear previous geocoding attempts if retry is True
         if retry:
             sql = """UPDATE  ag_login
-                     SET latitude = %s, longitude = %s, elevation = %s, cannot_geocode = %s
+                     SET latitude = %s,
+                     longitude = %s,
```

Comment:
these need to line up with latitude actually. See the SQL guidelines in qiita, which I will port over to here soon.